Assets/Scripts/Grid/TerrainMod.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;
+using System;
+
+public static class TerrainMod {
```

Comment:
First instance I'm noticing this, but maybe add a detailed description of the class' purpose at the top for all non-unity classes.